### ISSUE TITLE
Custom peatio and barong container with preinstalled editor and debug tools

### ISSUE THREAD

**Issue by ysv** [rubykube/workbench#185]

**Comment by shal** [comment]
@ysv Are you sure this issue should be in workbench?

**Comment by ysv** [comment]
@shal yes because each time I exec into patio container and want to debug something I need to update && upgrade && install

**Comment by shal** [comment]
I think it is not related to workbench, is we don't store Dockerfiles

**Comment by ysv** [comment]
`make install-tools` -> `docker run -it apt-get update ...`

**Comment by ysv** [comment]
or we can use custom dockerfiles for this purpose

**Comment by shal** [comment]
@ysv I see the point, but I think @imarakhovskiy misunderstood your task.

https://github.com/rubykube/peatio/blob/fb67d405433de5432562294c2cf4b6d3d9ecd76f/Dockerfile#L35-L37

**Comment by ysv** [comment]
Totally 😅
